src/Homeworks/Homework5_1/Routers/MaxTreeBuilder.cs

```diff
@@ -0,0 +1,98 @@
+namespace Routers
```

Comment:
```suggestion
namespace Routers;
```

И дальше без фигурной скобки. Хорошая практика, file-scoped неймспейсы